### PR TITLE
🐛 Return absolute URLs for search results

### PR DESCRIPTION
**Pull request by matthiasrohmer**
... making them relative was only to have better DX while testing.

### CODE REVIEW

File: platform/lib/routers/search.js [hunk 1]
```diff
@@ -114,9 +114,13 @@ function createPageObject(csePageItem) {
     url: csePageItem.link,
   };
 
-  const hostMatch = page.url.match(REMOVE_HOST_PATTERN);
-  if (hostMatch) {
-    page.url = page.url.substring(hostMatch[0].length);
+  // For better DX while testing the search make links relative
+  // to point to local documents
+  if (config.isDevMode() && config.isLocalMode()) {
```
Comment (sebastianbenz) [inline platform/lib/routers/search.js]: should this be `||`?

Comment (matthiasrohmer) [reply platform/lib/routers/search.js]: 🤦‍♂Thanks.

**Comment by westonruter** [comment]
When will this be deployed into production?

**Comment by sebastianbenz** [comment]
It's already live in prod.

>

**Comment by westonruter** [comment]
@sebastianbenz OK, I can see it for the `/search/do` endpoint but not for `/search/highlights`. See https://amp.dev/search/highlights?locale=en

Relative URLs are still being served.

Also, for `/search/do` not all URLs are absolute yet. Namely, `nextUrl` and `exampleUrl` are missing the host.

**Pull request merged**